pkg/controller/render/render_controller.go

```diff
@@ -165,6 +165,8 @@ func (ctrl *Controller) deleteMachineConfigPool(obj interface{}) {
 
 func (ctrl *Controller) addMachineConfig(obj interface{}) {
 	mc := obj.(*mcfgv1.MachineConfig)
+	glog.Infof("MachineConfig %s added", mc.Name)
```

Comment:
Not superconfident but wondering if moving log statement here may lead to false positives logs. Perhaps we can keep the earlier logging while removing verbosity.

Reply:
I think we won't have this issue as, if we got here we were "informed" that an add action has been performed at the api level (same for update/delete) so it should always be safe to log right?

Reply:
great!